.circleci/scripts/python_doc_push_script.sh

```diff
@@ -60,6 +60,8 @@ pip install -q https://s3.amazonaws.com/ossci-linux/wheels/tensorboard-1.14.0a0-
 # Get all the documentation sources, put them in one place
 pushd "$pt_checkout"
 git clone https://github.com/pytorch/vision
+git fetch --tags --all
+git checkout v0.5.0
```

Comment:
This should come after "pushd vision" because we're not in the vision directory yet

Reply:
oh good catch!